libraries/browser-tracker-core/src/tracker/out_queue.ts

```diff
@@ -222,7 +225,7 @@ export function OutQueueManager(
   }
 
   const postable = (queue: Array<PostEvent> | Array<string>): queue is Array<PostEvent> => {
-    return typeof queue[0] === 'object';
+    return typeof queue[0] === 'object' && 'evt' in queue[0];
```

Comment:
This is probably no longer necessary:

```suggestion
    return typeof queue[0] === 'object';
```

Reply:
I added that as an extra sanity check of the POST type requests.